samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/jsf/renderer/TimerBarRenderer.java

```diff
@@ -97,8 +97,8 @@ public void encodeEnd(FacesContext context, UIComponent component)
         }
 
          String contextPath = context.getExternalContext().getRequestContextPath();
-         writer.write("\n<script type=\"text/javascript\" src=\"" +
-           contextPath + SCRIPT_PATH + "timerbar.js\"></script>");
+         writer.write("\n<script type=\"text/javascript\" src=\"/library/webjars/momentjs/2.29.1/min/moment-with-locales.min.js\"></script>");
```

Comment:
@bjones86 for this would be preferred
<script>includeWebjarLibrary("momentjs");</script>

makes it future proof

Reply:
Thanks, that works!